### PR TITLE
Correct forms labels

### LINKED ISSUE #214: Correct forms labels
**Describe the bug**
- Need to add format for the date label.
- some of the forms labels were shortcuts, need to correct them.
examples - 
`min` - `Minimum`
`num` - `Number`

**To Reproduce**
Go to any form.

**Expected behavior**
correct the labels.

### PR DESCRIPTION
**Pull request by AmitAharoni**
# Description
- Added format for the date label - [YYYY-MM-DD].
- some of the forms labels were shortcuts, corrected them.
examples:
`min` - `Minimum`.
`num` - `Number`.

## Screenshots
image

### Issues closed by this PR
fix: #214.

### PR Dependencies
None.

### CODE REVIEW

**Comment by tamirho** [comment]
Nice.. can you do it in the other forms (in the apartments/seekers apps) as well?

File: users/forms.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 class UserCreationForm(forms.ModelForm):
     password1 = forms.CharField(label="Password", widget=forms.PasswordInput())
     password2 = forms.CharField(label="Password confirmation", widget=forms.PasswordInput())
+    birth_date = forms.DateField(label='Birthday')
```
Comment (Micha546) [inline users/forms.py:8]: I think that writing it this way has some other side effects, you can change a label of a field of a modelForm by writing 
`labels={
             'birth_date': 'Birthday'
}`
or something like that

Comment (Micha546) [reply users/forms.py:8]: https://stackoverflow.com/questions/36905060/how-can-i-change-the-modelform-label-and-give-it-a-custom-name/36905090

Comment (DanielMy777) [reply users/forms.py:8]: I don't see it causing any harm.
If this works there's no reason to update it IMO.

Comment (Micha546) [reply users/forms.py:8]: I think that it's better to write it the way shown in the stackoverflow page, it's a small change too 🧙

Comment (Micha546) [reply users/forms.py:8]: But it doesn't really matter if it works

**Review by DanielMy777 — APPROVED** [review]

File: users/forms.py (hunk 1 above)
Comment (AmitAharoni) [reply users/forms.py:8]: It's really doesn't matter. both doing the same.

**Comment by AmitAharoni** [comment]
> Nice.. can you do it in the other forms (in the apartments/seekers apps) as well?
@tamirho 
I did in the  `user` form which both `seekers` and `apartments` app using. 
So they get the same result :)

**Pull request merged**